apps/et/et-ccd-case-migration/demo-image-policy.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: image.toolkit.fluxcd.io/v1beta1
```

Comment:
this file needs to be added into the automation folder if it's not already there

Reply:
done